handbook/singularity.md

```diff
@@ -0,0 +1,40 @@
+# Singularity
+
+On occasion, the operating system and Rstudio installations that are available via [FASRC VDI Apps](https://docs.rc.fas.harvard.edu/kb/vdi-apps/) are not compatible with some R packages. A viable way to surmount package incompatibility issues is using docker containers. The [Rocker Project](https://rocker-project.org/images/) images run R and Rstudio containerized installations that come in various flavors. Three of the most popular rocker image builds are: `rocker/tidyverse`, `rocker/verse`, and `rocker/geospatial`. The official overview of these images is found [here](https://rocker-project.org/images/versioned/rstudio.html#overview). 
+
+## Downloading a rocker/docker image with singularity
+
+To work on a containerized Rstudio in FASSE, first identify the rocker distribution that best fits your needs. For the purpose of this tutorial, we will work with `rocker/geospatial:3.6.0`.
+
+Once logged in to FASSE, create an `images` folder in your user directory and run
+
+```
+cd images
+singularity pull docker://rocker/geospatial:3.6.0
+```
+
+Once the download has completed you should be able to see the file `geospatial_3.6.0.sif`.
```

Comment:
If you've tried using several containers, it can also be helpful to run "singularity cache list" which allows you to see any containers that you have

Reply:
And "singularity cache clean" removes everything in the cache if you need to redo things

Reply:
Cool! That is something I was unsure about singularity, whether containers are undistinguishable and whether containers stop running after you exit the shell

Reply:
These is quite nice, I tried it and I didn't know I had that cache occupying space. Including in the next version.